Source/Core/DolphinQt/Settings/GameCubePane.cpp

```diff
@@ -186,12 +189,27 @@ void GameCubePane::OnConfigPressed(int slot)
   {
     bool ok;
     const auto new_mac = QInputDialog::getText(
+        // i18n: TAP is a type of virtual network interface, MAC stands for Media Access Control
```

Comment:
Since you removed the "TAP" from the string, it need not be in the translation comment I suppose.